ui/app/routes/vault/cluster/access/method/item/list.js

```diff
@@ -9,14 +9,14 @@ export default Route.extend(ListRoute, {
 
   getMethodAndModelInfo() {
     const { item_type: itemType } = this.paramsFor('vault.cluster.access.method.item');
-    const { path: method } = this.paramsFor('vault.cluster.access.method');
+    const { path: id } = this.paramsFor('vault.cluster.access.method');
```

Comment:
I think there's a couple of models here, would it make sense to call this the `methodId` or `methodPath` or even `authMethodPath` if you want extra verbose ? Is this essentially the mount path for the auth method?

Reply:
that's correct -- it's the mount path for the auth method. i had been struggling with naming since it's really easy to confuse the model, mountType, and path. thanks for the suggestion!